tasks/apply-mapping/apply-mapping.yaml

```diff
@@ -123,7 +123,7 @@ spec:
                   exit 1
                 fi
 
-                translated_tags="$(jq -c --arg tag "$tag" '. + [$tag]' <<< "$translated_tags")"
+                translated_tags="$(jq -c --arg tag "$tag" '. + [$tag] | unique' <<< "$translated_tags")"
```

Comment:
As Happy pointed out, the tests failed. It turns out that [`uniq` also sorts the items](https://jqlang.github.io/jq/manual/#unique-unique_by) as a side effect. 

```
[check-result : check-result] + test '["123456","defaultTag","foo-123456","tag-labelvalue","tag1-2024-07-29","tag2-2024-07-29","tag3-2024-07-29-testrev","testrev","testrev-bar","testrevision","testrevision-abc"]' == '["defaultTag","foo-123456","tag-labelvalue","tag1-2024-07-29","tag2-2024-07-29","tag3-2024-07-29-testrev","123456","testrevision-abc","testrev-bar","testrevision","testrev"]'

  Pipelinerun test-apply-mapping-tag-expansion failed
```

So the question is: Do we care? Is this ok? Probably yes? @johnbieren WDYT?

Reply:
nvm, I solved it another way and now the order stays.